### ISSUE TITLE
Setting default values for query params [Question]

### ISSUE THREAD

**Issue by jyjyj** [kobylynskyi/graphql-java-codegen#72]
Hi, how can I set the default values for query params? In my graphql schema, I'm writing something like:

`getCurrentAlerts(size: Int=10, skip: Int=0, showAll: Boolean=false): [Alert!]!`

But in the generated file, I'm simply getting:

`public interface GetCurrentAlertsQuery {
    @javax.validation.constraints.NotNull
    Collection<Alert> getCurrentAlerts(Integer size, Integer skip, Boolean showAll) throws Exception;
}`

Thanks!

**Comment by joffrey-bion** [comment]
Out of curiosity, what would you expect the generated interface to look like?

I believe the problem of the default values rather lies on the other side: the caller of this method.
Are you using [GraphQL java tools](https://www.graphql-java-kickstart.com/tools/)?

**Comment by kobylynskyi** [comment]
Closing the issue as there is no way to specify default values in the generated interfaces.
@jyjyj if you have any suggestions, please let me know.

**Comment by jyjyj** [comment]
Hi @joffrey-bion and @kobylynskyi, I have just checked further and I noticed that the default values are indeed being fetched (I'm using DataFetcher to read the schema files to generate the GraphQL object), and I am able to propagate these values to my implementations. I suppose then that I should expect the default values to come from the graphql schema side rather than the interfaces themselves? Either way, there is no issue now. Thanks!
